src/main/java/at/hannibal2/skyhanni/utils/TimeLimitedSet.kt

```diff
@@ -26,13 +27,15 @@ class TimeLimitedSet<T : Any>(
 
     fun remove(element: T) = cache.remove(element)
 
-    operator fun minusAssign(element: T) = remove(element)
+    operator fun minusAssign(element: T) {
+        remove(element)
+    }
 
     operator fun contains(element: T): Boolean = cache.containsKey(element)
 
     fun clear() = cache.clear()
 
-    fun toSet(): Set<T> = HashSet(cache.keys())
+    fun toSet(): MutableSet<T> = cache.keys
```

Comment:
im not sure i like the idea of exposing mutable cache outside this class. this allows unintended behaviour